### DISCUSSION TITLE
streams not recognized after server reset

### DISCUSSION THREAD

**Discussion by DeanShandlerSL** [ant-media/Ant-Media-Server#5196] [Q&A]
I have experienced this issue multiple times where the Antmedia server fails to recognize streams after the PC hosting Antmedia is disconnected from and then reconnected to the power supply. 

Although the streams are visible in the attached picture, upon restarting, the green arrows indicating their availability are missing. 
To resolve this, I have found a workaround by clicking the pencil icon and then clicking OK, which prompts the server to search for the streams and make them available again.

However, it is important to note that the mentioned workaround is not an acceptable long-term solution, as it requires manual intervention each time the power supply is disconnected. This inconsistency and the need for manual intervention hinder our ability to work effectively with the Antmedia server. Therefore, it is crucial to find a more reliable and automated solution to ensure that the server consistently recognizes the streams without any manual intervention.

<img width="1485" alt="Screenshot 2023-06-05 at 9 50 38" src="https://github.com/ant-media/Ant-Media-Server/assets/89450090/78afe0f1-97c0-4750-8451-041245e8efa6">

**Comment by mekya** [comment]
Hi @DeanShandlerSL,
As I far as I understand, you've IP cameras connecting to the server with RTSP and when Ant Media Server is killed and started, it does not recover them. It just shows the wrong state(Broadcasting) on the web panel. 

Even if Ant Media Server should have recovered the streams automatically when restarting, it seems that this functionality has an issue and we're lacking integration tests for this scenario. 

 Lastly, are you using them in cluster mode or standalone mode?
